src/foundry/common/utils/helpers.d.mts

```diff
@@ -304,7 +304,11 @@ export function isNewerVersion(v1: number | string, v0: number | string): boolea
  * @param value - The value to test
  * @returns Is the value empty-like?
  */
-export function isEmpty(value: undefined | null | unknown[] | object | Set<unknown> | Map<unknown, unknown>): boolean;
+
+export function isEmpty(
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
+  value: undefined | null | unknown[] | object | Set<unknown> | Map<unknown, unknown> | unknown,
```

Comment:
Using `unknown` will collapse, even in intellisense, to just `unknown`.

I recommended `undefined | null | AnyArray | AnyObject | Set<unknown> | Map<unknown, unknown> | NonNullish` because it doesn't collapse.